src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -105,17 +105,19 @@ export default class ArrayField extends React.Component {
   }
 
   scrollToRow(id) {
-    setTimeout(() => {
-      scroller.scrollTo(
-        `table_${id}`,
-        window.Forms?.scroll || {
-          duration: 500,
-          delay: 0,
-          smooth: true,
-          offset: 0,
-        },
-      );
-    }, 100);
+    if (!this.props.uiSchema['ui:options'].doNotScrollOnAdd) {
```

Comment:
We do want to the same functionality on the review page as well and for the edit function as well.

Reply:
Do you have a concern about the review/edit not scrolling?

Reply:
My point about `handleEdit`/`handleAdd` is that `doNotScrollOnAdd` doesn't suggest that it will _also_ be applied when an edit happens.

And since you want the same functionality on the review page, I was asking if you had tested/looked at it, since I didn't see any changes to the review version of `ArrayField`.

https://github.com/department-of-veterans-affairs/vets-website/blob/e0a7209f786720c49799eec64ad07a09840aa177/src/platform/forms-system/src/js/review/ArrayField.jsx#L90-L102

Reply:
Ah! I that makes sense I will rename the flag, as well as check the review. Do you have any guidance on testing for scrolling? I was going to create some cypress tests for it.

Reply:
Just to circle around on this. I looked at the review instance and it does use that [component you linked](https://github.com/department-of-veterans-affairs/vets-website/blob/e0a7209f786720c49799eec64ad07a09840aa177/src/platform/forms-system/src/js/review/ArrayField.jsx#L90-L102) and I do not think I have configured anything _not_ to use it.

Reply:
The only guidance for testing scrolling is to not do it in cypress :upside_down_face: . Testing that kind of thing in cypress would lead to very brittle tests, and if scrolling testing is necessary then it would be better off as part of a unit test (i.e. making sure that the right function gets called with the right argument).

Reply:
I added a _very simple_ unit test, checking for props. If you want any more let me know, but it's ready to go from my perspective.

Reply:
Thank you for your feedback and work! :-)

Reply:
Have you tested this on the review page?

Reply:
Yes I have tested it on the review page